sys/netpfil/ipfilter/netinet/ip_proxy.c

```diff
@@ -685,7 +685,7 @@ ipf_proxy_ioctl(ipf_main_softc_t *softc, caddr_t data, ioctlcmd_t cmd,
 	caddr_t ptr;
 	int error;
 
-	mode = mode;	/* LINT */
+	(void)mode;	/* UNUSED */
```

Comment:
There is an __unused ctypes.h annoation that we COULD put on the args, but given the age of this code, this construct is fine, imho.

Reply:
We should use __unused.

I converted it from K&R a number of years ago. __unused is the next progression.